climakitae/uncertain_utils.py

```diff
@@ -325,6 +361,80 @@ def grab_temp_data(copt):
 
     return mdls_ds
 
+## Grab data - internal variability analysis
+def grab_ensemble_data(copt,cmip_names,alpha_sort=True):
+    """Returns processed data from multiple CMIP6 models for uncertainty analysis.
+
+    Searches the CMIP6 data catalog for data from models that have specific
+    ensemble member id in the historical and ssp370 runs. Preprocessing includes
+    subsetting for specific location and dropping the member_id for easier
+    analysis.
+
+    Attributes
+    ----------
+    copt: object
+        Selections: variable, area_subset, location, area_average, timescale
+    alpha_sort: bool
+        Set to True if sorting model names alphabetically is desired
+    cmip_names: list
+        Specific models
+
+    Returns
+    -------
+    List
+        Historical (1981-2010) CMIP6 xr.Datasets concatenated along member_id
+        for each specified model
+    List
+        Warming level CMIP6 xr.Datasets w/ multiple member_ids
+        for each specified model
```

Comment:
This should be lowercase `list of xr.Dataset` instead of `List`, with the "l" being lowercase

Reply:
wait, what? this has since been changed... please go to the latest commit

Reply:
sorry about that!